src/agent/rustjail/src/container.rs

```diff
@@ -995,9 +995,9 @@ impl BaseContainer for LinuxContainer {
 
         info!(logger, "entered namespaces!");
 
-        self.created = SystemTime::now();
-
         if p.init {
+            self.created = SystemTime::now();
+
```

Comment:
Hi @Tim-Zhang - I'm not sure this is correct due to:

https://github.com/kata-containers/kata-containers/blob/main/src/agent/rustjail/src/container.rs#L1425

Which looks like for the non-init case we need to unset `self.created`. If that's true, we should change `self.created` to `Option<SystemTime>` though and set it to `None` initially.

Reply:
yes, it is indeed a problem, let me have a think

Reply:
This code is quite subtle, so when we have a fix, it would be great to add some unit tests if possible.

Reply:
@jodh-intel I have deleted this line, now we use the time when the LinuxContainer object was created as the `created` time. I think is good enough because the init status is `ContainerState::Created`, so we use set the created-time in the real-time makes sense.

And since we don't change the `created` time, the unit test can be omitted.